src/iso19111/c_api.cpp

```diff
@@ -1204,9 +1204,9 @@ const char *proj_as_wkt(PJ_CONTEXT *ctx, const PJ *obj, PJ_WKT_TYPE type,
  * @param type PROJ String version.
  * @param options NULL-terminated list of strings with "KEY=VALUE" format. or
  * NULL.
- * The currently recognized option is USE_ETMERC=YES to use
- * +proj=etmerc instead of +proj=tmerc (or USE_ETMERC=NO to disable implicit
- * use of etmerc by utm conversions)
+ * The currently recognized option is USE_STMERC=YES to use
+ * +proj=stmerc instead of +proj=tmerc (or USE_STMERC=NO to disable implicit
```

Comment:
The part "(or USE_STMERC=NO to disable implicit use of tmerc by utm conversions)" doesn't make sense. We probably no longer need a specific behaviour for USE_STMERC=NO